jpo-geojsonconverter/pom.xml

```diff
@@ -90,26 +90,37 @@
         </dependency>
 
         <!-- Kafka Dependencies -->
+        <dependency>
+            <groupId>org.springframework.kafka</groupId>
+            <artifactId>spring-kafka</artifactId>
+            <version>2.8.11</version>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.kafka</groupId>
+            <artifactId>spring-kafka-test</artifactId>
+            <version>2.8.11</version>
+            <scope>test</scope>
+        </dependency>
         <dependency>
             <groupId>org.apache.kafka</groupId>
             <artifactId>kafka-streams-test-utils</artifactId>
-            <version>3.2.3</version>
+            <version>3.1.2</version>
```

Comment:
Why downgrade the streams version?

Reply:
I had some issues with compatibility of kafka dependencies and referenced the spring-boot compatibility versions for the kafka imports.
https://docs.spring.io/spring-boot/docs/2.7.6/reference/html/dependency-versions.html